test/rekt/container_source_test.go

```diff
@@ -53,7 +53,6 @@ func TestContainerSourceWithSinkURI(t *testing.T) {
 		k8s.WithEventListener,
 		environment.Managed(t),
 	)
-	t.Cleanup(env.Finish)
```

Comment:
I had copy-pasted these not realizing that this is what `environment.Managed(t)` does automatically